internal/postprocessor/main.go

```diff
@@ -327,8 +334,12 @@ func (c *config) getPR(ctx context.Context) (*github.PullRequest, error) {
 		return nil, err
 	}
 	var owlbotPR *github.PullRequest
+	branch := c.branchOverride
```

Comment:
One option would be to move some logic so that the config has one branch field. When the config is first created it checks to see if the branchOverride flag was passed, if so it sets the branch field to the override, otherwise it sets it to the owl-bot-copy branch.